packages/cli/src/__tests__/commands/devices.test.ts

```diff
@@ -346,4 +348,55 @@ describe('DevicesCommand', () => {
 		expect(getSpy).toBeCalledWith(deviceId, { includeStatus: undefined })
 		expect(getHealthSpy).toBeCalledWith(deviceId)
 	})
+
+	it('includes room and location names when verbose flag is set', async () => {
+		const device = { deviceId: 'device-id' } as Device
+		const deviceWithRoom = { ...device, location: 'location-name', room: 'room-name' } as Device & WithNamedRoom
+		getSpy.mockResolvedValueOnce(device)
+		withLocationAndRoomMock.mockResolvedValueOnce(deviceWithRoom)
+
+		await expect(DevicesCommand.run([deviceId, '--verbose'])).resolves.not.toThrow()
+		expect(outputItemOrListMock).toHaveBeenCalledTimes(1)
+		expect(outputItemOrListMock).toBeCalledWith(
+			expect.anything(),
+			expect.anything(),
+			deviceId,
+			expect.anything(),
+			expect.anything(),
+		)
+
+		const getFunction = outputItemOrListMock.mock.calls[0][4]
+
+		await expect(getFunction(deviceId)).resolves.toBe(deviceWithRoom)
+		expect(getSpy).toBeCalledWith(deviceId, { includeStatus: undefined })
```

Comment:
Not super important but I think it would be worth it to also expect that each of these have been called exactly once. (Same in next test.)

Reply:
Done.